include/vssp.h

```diff
@@ -413,9 +413,10 @@ class VsspDriver
                 else if (lines[0].compare("GET:tblh") == 0)
                 {
                   boost::shared_array<double> tbl_h(new double[cells.size()]);
+                  int i = 0;
                   for (auto &cell : cells)
                   {
-                    tbl_h[i] = static_cast<double>(std::strtol(cell.c_str(), nullptr, 16) / 65535);
```

Comment:
This had to be `static_cast<double>(std::strtol(cell.c_str(), nullptr, 16)) / 65535` or `std::strtol(cell.c_str(), nullptr, 16) / 65535.0`, otherwise all of the `tbl_h` is calculated in integer precision.